src/sbl_filing_api/services/submission_processor.py

```diff
@@ -126,3 +125,22 @@ async def validate_and_update_submission(period_code: str, lei: str, submission:
         )
         submission.state = SubmissionState.VALIDATION_ERROR
         await update_submission(submission)
+
+
+def build_validation_results(result):
+    val_json = json.loads(df_to_json(result[1]))
+
+    if result[2] == ValidationPhase.SYNTACTICAL.value:
+        val_res = {"syntax_errors": {"count": 0, "details": []}}
+        val_res["syntax_errors"]["details"] = val_json
+        val_res["syntax_errors"]["count"] = len(val_res["syntax_errors"]["details"])
```

Comment:
Just like below, this section can be shortened to jsut:
val_res = {"syntax_errors": {"count": len(val_json), "details": val_json}}